bin/build-jq.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -ex
+
+pushd bin
+curl --location https://github.com/stedolan/jq/releases/download/jq-1.5/jq-1.5.tar.gz >jq-1.5.tar.gz
+tar xvf jq-1.5.tar.gz
+cd jq-1.5
+./configure --disable-maintainer-mode && make
+mv jq ..
+popd
+
```

Comment:
Building a newer version of jq from source to support Windows machines. See https://github.com/exercism/xjava/pull/277 for context.